src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -182,9 +182,9 @@ public PersistentDataStore() {
         int start = Integer.parseInt((String) entity.getProperty("start"));
         int end = Integer.parseInt((String) entity.getProperty("end"));
         Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
-        Mention mention =
-            new Mention(
-                uuid, userWhoWasMentioned, userWhoDidTheMentioning, start, end, creationTime);
+        //Message message = Message.parse((String) entity.getProperty("message"));
+        // TODO: Get message to create mention
+        Mention mention = new Mention(uuid, userWhoWasMentioned, userWhoDidTheMentioning, start, end, creationTime, null);
```

Comment:
that trailing null is a little scary. What's up with that?  (If it's a field that hasn't been persisted, then we need to add it to the persistence logic.)